modules/core/util/include/util.hpp

```diff
@@ -27,6 +26,46 @@ using NlohmannJsonTypeError = nlohmann::json::type_error;
 
 namespace ppc::util {
 
+/**
```

Comment:
Pretty signature is an overkill for our task. `__func__` is concise and crossplatform